app/Http/Controllers/WaitingRoomsController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Gladiator\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Gladiator\Http\Controllers\Controller;
+use Gladiator\Models\WaitingRoom;
+
+class WaitingRoomsController extends Controller
+{
+    private $validationRules = [
+        'campaign_id' => 'required|numeric',
+        'campaign_run_id' => 'required|numeric',
+        'signup_start_date' => 'required|date',
+        'signup_end_date' => 'required|date'
```

Comment:
Same with Joe's competition crud, might be nice to pull these out into a Form Request to help keep things cleaner but totally not needed for merge :)